selfdrive/car/volkswagen/carcontroller.py

```diff
@@ -13,7 +13,8 @@ def __init__(self, dbc_name, CP, VM):
     self.apply_steer_last = 0
     self.frame = 0
 
-    self.packer_pt = CANPacker(DBC_FILES.mqb)
+    self.packer_pt = CANPacker(dbc_name)
+    self.CCP, self.CCS = MQBCarControllerParams, mqbcan
```

Comment:
break these up into two lines

Reply:
also note that these abstractions only happen to work because MQB and PQ are so similar. if they start to diverge more for any reason, then you'll need to develop this abstraction a bit more

Reply:
MQB vs PQ is the very hardest split we have, and are likely to have for the forseeable future.

We have lateral-only control of at least one [NSF](https://en.wikipedia.org/wiki/Volkswagen_Group_New_Small_Family_platform) which turns out to be indistinguishable from PQ except the wheel-speed sensor signal is missing, because (this is not a joke) some of them have no wheel speed sensors. The community A5/S5 port for MLB (pre-Flexray) looks like early/modified MQB on the wire and uses the same J533 harness. MQBevo (Golf Mk8 and friends) will look like MQB with specialsauce. MEB (ID.3/ID.4 and friends), if we can make a lateral-only port work, will look like a MQBevo variant.

That accounts for almost everything that we could conceivably drive except for MLBevo/Flexray.